src/python/dxpy/scripts/dx_build_app.py

```diff
@@ -576,6 +576,10 @@ def _build_app_remote(mode, src_dir, publish=False, destination_override=None,
     else:
         builder_app = 'app-tarball_applet_builder'
 
+    app_spec = _parse_app_spec(src_dir)
+    if 'distribution' in app_spec['runSpec'] and 'release' in app_spec['runSpec'] and app_spec['runSpec']['release'] == '14.04':
```

Comment:
Seems like `distribution` is presumed to be "Ubuntu", so it should be legal to just write `{"release": "14.04", ...}` (without `distribution`) in your `runSpec` and have it be treated the same way.